models/QtiRunnerInitDataBuilder.php

```diff
@@ -149,13 +166,17 @@ protected function getTestMap(QtiRunnerServiceContext $context)
                     $itemData = $this->qtiRunnerService->getItemData($context, $item->getHref());
 
                     $itemId = $item->getIdentifier();
+                    $score = $itemsStates[$itemId]['score'] ?: [];
+
                     $items[$itemId] = [
                         'id' => $itemId,
                         'label' => $itemData['data']['attributes']['label'],
                         'position' => $position,
                         'categories' => [],
-                        'score' => 0,
-                        'maxScore' => 0
+                        'informational' => false,
+                        'skipped' => false,
+                        'score' => array_sum($score),
+                        'maxScore' => count($score)
```

Comment:
Not sure this is the proper way to compute the score and the maxScore, as this will introduce wrong results.

You should rather rely on the `SCORE` and `MAXSCORE` variable of each item.
If no correct response is set in the interactions, the `MAXSCORE` variable won't be created, and therefore you should reflect that by either a `0` or a `null`.

Reply:
There is also something wrong in the item responses, as they are empty now in the frontend. This is because the format of the `testResponses` has changed in the returned data.

Pay also attention to the name of the git branch, it does not comply with the rules: it should start with the kind of change (`fix`, `feature`, ...). Usually the template is: `type/jira-ticket/short-description`

Reply:
Is it required to use gitflow? And where to check the rules?

Reply:
The rules are there: https://oat-sa.atlassian.net/wiki/spaces/DEV/pages/11338720/Development+Workflow+with+Github